.github/workflows/basic.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       # {{SQL CARBON TODO}} Update node modules caching to work with our cache keys
```

Comment:
Remove this comment too - IIRC I just never got around to actually verifying if this worked. So hopefully this is all that's required, although you may want to still check to see if VS Code has made any other changes in the meantime while we're making changes here.